app/views/tag/show.html.erb

```diff
@@ -28,7 +28,9 @@
 
   <%= render partial: "tag/show/questions" %>
   <%= render partial: "tag/show/related_tags" %>
-  <p> <%= render partial: 'tag/subscribe_button', locals:{tags: @related_tags } %></p>
+  <% if !@related_tags.nil? %>
```

Comment:
You've done a really good job here, @shubhi2000. One tip, though. With Ruby, you don't need to make an explicit check for `nil`, especially when, in your case, you're checking to see if a variable is not nil. To check if a variable is not nil, you can just do `if variable`, which will return true if it's not nil, or false if it's nil.

So, you can write your code like this:

```suggestion
  <% if @related_tags %>
```
It makes it a lot less complex to read.

Reply:
@madeofhuman thank you so much. I am completely new to Ruby, that's why.

Reply:
It's perfectly all right. 😄 I remember when someone gave me this same tip on my first Ruby project.

Reply:
Ohh, nice.
It is really helpful when someone gives us a tip, it benefits us in the long-term. Thanks!